cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/statements/expressions/CastExpression.java

```diff
@@ -97,19 +98,19 @@ public void possibleSubTypesChanged(HasType src, List<HasType> root) {
   public void setCastOperator(int operatorCode) {
     switch (operatorCode) {
       case 0:
-        setName("cast");
+        setFullName(new Name("cast", null, this.getLanguage().getNamespaceDelimiter()));
```

Comment:
probably would make more sense to set a variable with the name and then use this variablle directly in newXXX